shell/platform/windows/accessibility_bridge_windows.h

```diff
@@ -2,59 +2,59 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_ACCESSIBILITY_BRIDGE_DELEGATE_H_
-#define FLUTTER_SHELL_PLATFORM_WINDOWS_ACCESSIBILITY_BRIDGE_DELEGATE_H_
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_ACCESSIBILITY_BRIDGE_WINDOWS_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_ACCESSIBILITY_BRIDGE_WINDOWS_H_
 
 #include "flutter/shell/platform/common/accessibility_bridge.h"
 
-#include "flutter/shell/platform/windows/flutter_platform_node_delegate_windows.h"
 #include "flutter/shell/platform/windows/flutter_windows_engine.h"
+#include "flutter/shell/platform/windows/flutter_windows_view.h"
 
 namespace flutter {
 
 class FlutterWindowsEngine;
+class FlutterPlatformNodeDelegateWindows;
 
-// The Win32 implementation of AccessibilityBridgeDelegate.
-//
-// Handles requests from the accessibility bridge to interact with Windows
-// accessibility APIs. This includes routing accessibility events fired from
-// the framework to Windows, routing native Windows accessibility events to the
-// framework, and creating Windows-specific FlutterPlatformNodeDelegate objects
-// for each node in the semantics tree.
-class AccessibilityBridgeDelegateWindows
-    : public AccessibilityBridge::AccessibilityBridgeDelegate {
+// The Win32 implementation of AccessibilityBridge.
```

Comment:
Consider keeping the rest of the documentation from before:

```suggestion
// This interacts with Windows accessibility APIs, which includes routing
// accessibility events fired from the framework to Windows, routing native
// Windows accessibility events to the framework, and creating Windows-specific
// FlutterPlatformNodeDelegate objects for each node in the semantics tree.
```

Reply:
Thank you. I'm copying (and adapting) this paragraph to the mac bridge as well.